apps/xero/queue.py

```
@@ -126,6 +127,26 @@ def __create_chain_and_run(workspace_id: int, xero_connection, chain_tasks: List
+    """
+    Handle common export scheduling logic for skip tracking, logging, posting skipped export summaries, and last export updates.
+    """
+    total_count = expense_groups.count()
+    last_export = (index + 1) == total_count
+
+    skip_reason = f"{error.repetition_count} repeated attempts" if error else "mapping errors"
+    logger.info(f"Skipping expense group {expense_group.id} due to {skip_reason}")
+    skip_export_count += 1
+    if triggered_by == ExpenseImportSourceEnum.DIRECT_EXPORT:
+        post_accounting_export_summary_for_skipped_exports(
+            expense_group, expense_group.workspace_id, is_mapping_error=False if error else True
```

Ruff (0.11.9)

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)